modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lar/PortletDataContextImpl.java

```diff
@@ -1241,7 +1241,8 @@ public String getZipEntryAsString(String path) {
 			return null;
 		}
 
-		return getZipReader().getEntryAsString(path);
+		return _zipEntriesCache.computeIfAbsent(
```

Comment:
@cleydyr I see the cache is populcated and read, but no one is going to remove/clear it, this may cause out of memory later.